dbt/include/athena/macros/materializations/models/table/create_table_as.sql

```diff
@@ -20,6 +20,7 @@
                                                  s3_data_naming,
                                                  external_location,
                                                  temporary) -%}
+  {%- set native_drop = config.get('native_drop', default='False') | as_bool -%}
```

Comment:
```suggestion
  {%- set native_drop = config.get('native_drop', 'False') | as_bool -%}
```
due to https://stackoverflow.com/questions/24463202/typeerror-get-takes-no-keyword-arguments